app/presenters/decidim/solutions/solution_presenter.rb

```diff
@@ -59,6 +59,78 @@ def description(links: false, extras: true, strip_tags: false)
         text
       end
 
+      # Render the solution objectives
```

Comment:
As we talked about in Slack, when PR #7 is reviewed/merged to master, this file will have conflict 'cause the changes already are on that PR. 
Just to keep in mind :wink:

Reply:
R #7 has already been merged